packages/extension/src/newtab/App.tsx

```diff
@@ -128,12 +109,16 @@ function InternalApp({
       : DEFAULT_TAB_TITLE;
   }, [unreadCount]);
 
-  console.log({ hostGranted });
+  if (!hostGranted) {
+    // eslint-disable-next-line no-param-reassign
+    pageRef.current = '/permissions';
```

Comment:
Not really your mistake, but we should probably use forwardRef.
We can log it as seperate.
@capJavert What do you think?

Reply:
Yeah but not crucial, agree it can be done separate 👍

Reply:
Will log it separately :)